projects/ccd-case-ui-toolkit/src/lib/shared/components/palette/case-file-view/components/case-file-view-folder/case-file-view-folder.component.ts

```diff
@@ -169,29 +210,6 @@ export class CaseFileViewFolderComponent implements OnInit, OnDestroy {
     }
   }
 
-  public sortDataSourceAscAlphabetically() {
-    const sortedData = this.nestedDataSource.map(item => {
-      item.sortChildrenAscending();
-
-      const newDocumentTreeNode = new DocumentTreeNode();
-      newDocumentTreeNode.name = item.name;
-      newDocumentTreeNode.type = item.type;
-      newDocumentTreeNode.children = item.children;
-
-      return newDocumentTreeNode;
-    });
-
-    this.updateNodeData(sortedData);
-  }
-  public sortDataSourceDescAlphabetically() {
-    const sortedData = this.nestedDataSource.map(item => {
-      item.sortChildrenDescending();
-      return item;
-    });
-
-    this.updateNodeData(sortedData);
-  }
-
   public updateNodeData(data: DocumentTreeNode[]) {
```

Comment:
@MateiHMCTS - consider setting a return type for this function, even if it is `void`